lib/util.js

```diff
@@ -155,11 +155,7 @@ const util = {
     autoGeneratedBraveToChromiumMapping[path.join(braveAppDir, 'brave_strings.grd')] = path.join(chromeAppDir, 'brave_strings.grd')
     autoGeneratedBraveToChromiumMapping[path.join(braveAppDir, 'settings_brave_strings.grdp')] = path.join(chromeAppDir, 'settings_brave_strings.grdp')
     autoGeneratedBraveToChromiumMapping[path.join(braveComponentsDir, 'components_brave_strings.grd')] = path.join(chromeComponentsDir, 'components_brave_strings.grd')
-    autoGeneratedBraveToChromiumMapping[path.join(braveComponentsDir, 'components_strings.grd')] = path.join(chromeComponentsDir, 'components_strings.grdp')
-    autoGeneratedBraveToChromiumMapping[path.join(braveComponentsDir, 'management_strings.grdp')] = path.join(chromeComponentsDir, 'management_strings.grdp')
-    autoGeneratedBraveToChromiumMapping[path.join(braveComponentsDir, 'password_manager_strings.grdp')] = path.join(chromeComponentsDir, 'password_manager_strings.grdp')
-    autoGeneratedBraveToChromiumMapping[path.join(braveComponentsDir, 'bookmark_bar_strings.grdp')] = path.join(chromeComponentsDir, 'bookmark_bar_strings.grdp')
-    autoGeneratedBraveToChromiumMapping[path.join(braveAndroidDir, 'java', 'strings', 'android_chrome_strings.grd')] = path.join(chromeAndroidDir, 'java', 'strings', 'android_chrome_strings.grd')
```

Comment:
Don't need to re-add here as `autoGeneratedBraveToChromiumMapping` is imported from l10nUtil.js